Private/DistributedIntegrationTest/config.dsc

```diff
@@ -21,5 +21,12 @@ config({
             isWritable: false,
             isReadable: true,
         },
+        {

```

Comment:
>{ [](start = 8, length = 1)

Why do we need a new mount here?

Reply:
My test case has a copyfile pip writing to this directory. If I don't mount it, it claims file wirte in unmounted directory and untrack it.